python/lsst/pipe/tasks/coaddInputRecorder.py

```diff
@@ -181,7 +181,7 @@ def __init__(self, *args, **kwargs):
         self.visitFilterKey = self.visitSchema.addField("filter", type=str, size=32,
                                                         doc="Filter associated with this visit.")
         self.ccdFilterKey = self.ccdSchema.addField("filter", type=str, size=32,
-                                                    doc="Filter associated with this visit.")
+                                                    doc="Filter (band) associated with this visit.")
```

Comment:
See above. The description should probably be synchronized with `self.visitFilterKey`.